### DISCUSSION TITLE
如何在 app.tsx 中的 layout 上面包一层我自己的组件

### DISCUSSION THREAD

**Discussion by CopyTheCode** [ant-design/ant-design#45830] [General]
大概需求就是在页面最顶部包一层我自己的组件，怎么实现，在线等

**Comment by Wxh16144** [comment]
大兄弟，不是你这样提问的，代码都没贴，框架也没说，这个一般人真回答不了...

**Reply by CopyTheCode** [reply]
用的是[Ant Design Pro]这个，在app.tsx里面直接导出layout 这个方法，这个方法可以配置顶部，侧边栏，内容区域，底部，现在想在页面的最顶层在包装一个自己的组件，应该怎么包装呢？

**Comment by qianweichen** [comment]
https://github.com/umijs/umi/discussions/11346

**Reply by qianweichen** [reply]
![image](https://github.com/user-attachments/assets/46956f53-d427-4c40-92ca-dd28d8d4c286)

**Reply by qianweichen** [reply]
![image](https://github.com/user-attachments/assets/51521b0f-491c-4dd7-bfb4-c7e75b59f806)
